static/js/components/dashboard/CoursePrice.js

```diff
@@ -29,7 +58,7 @@ export default class CoursePrice extends React.Component {
     }
 
     return <div className="course-price">
-      {price}
+      {price} {this.renderTooltip(course)}
```

Comment:
i think you should create a new function `courseTooltipText` which just returns the text for a course (or no text at all) and change `renderTooltip` to accept a `text` parameter. this `render` function (starting at line 46) would call `courseTooltipText`, then call `renderTooltip` if the text isn't blank. it's confusing to have `renderTooltip` being called in all cases, but in many cases it's not actually rendering anything